pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[tool.poetry]
+name = "alerticular"
+version = "0.0.1"
+description = "Handle incoming Alertmanager alerts and send them to Telegram"
+
+license = "AGPLv3+"
+
+authors = [
```

Comment:
This is a list and you started a bunch of PRs, so I guess your name is missing here ;)